### PR TITLE
feat : added single product screen

### PR DESCRIPTION
**Pull request by SanjayJatti**
- Added HTML And CSS for Single Product Page
single-product page

### CODE REVIEW

File: src/Components/SingleProductPage/SingleProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+export const SingleProduct = () => {
+  return (
+    <div>
+      <Header />
+      <div className="single-product-wrapper">
+        <div className="product-image">
+          <img
+            className="img-responsive"
+            src="https://rukminim2.flixcart.com/image/746/895/jvv9zm80/watch/y/x/8/g604-casio-original-imafgh4uj5ump4gn.jpeg?q=50"
```
Comment (t007rushi) [inline src/Components/SingleProductPage/SingleProduct.jsx:13]: use prettier

File: src/Components/SingleProductPage/SingleProduct.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <i class="fas fa-check-circle font-inherit margin-r-xs"></i>
+            Currently in stock
+          </p>
+          <div className="flex-row gap-md margin-t-lg">
+            <button className="btn btn-primary">
+              <i className="fas fa-shopping-cart font-inherit margin-r-xs"></i>
+              ADD TO CART
+            </button>
+            <button className="btn btn-outline-secondary">
+              {" "}
```
Comment (t007rushi) [inline src/Components/SingleProductPage/SingleProduct.jsx:49]: remove spaces

**Review by t007rushi — APPROVED** [review]
some minor suggestions else looks good

**Review by prathmeshjagtap — APPROVED** [review]
code is well structured ,
and you have  followed all CSS rules

**Pull request merged**